requirements.txt

```diff
@@ -1,16 +1,21 @@
 boto==2.48.0
 certifi==2018.4.16
 chardet==3.0.4
+coverage==4.5.1
+coveralls==1.3.0
 cssselect==1.0.3
 cssutils==1.0.2
 Django==1.10
 django-appconf==1.0.2
 django-compressor==2.1
 django-ses-backend==0.1.1
 djmail==1.0.1
+docopt==0.6.2
 futures==3.1.1
 idna==2.6
 lxml==4.2.1
+mock==2.0.0
+pbr==4.0.2
```

Comment:
What are you using this package for?

Reply:
I remember we installed mock because @rogerokello needed it to run his tests. For the case of pbr and docopt, I believe they were installed probably with other dependencies, I never installed them individually.

Reply:
I am referring to `pbr==4.0.2`